cli/cmd/Environment.go

```diff
@@ -0,0 +1,131 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/nais/naisd/api"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var environmentCommand = &cobra.Command{
+	Short: "Update environment variables",
+	Long:  `Command to update environment variables with values fetched from the defined fasit resource`,
+	Use:   `env [flags] <application_name>`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			cmd.Help()
+			os.Exit(1)
+		}
+
+		application := args[0]
+		username := os.Getenv("FASIT_USERNAME")
+		password := os.Getenv("FASIT_PASSWORD")
+
+		inline := true
+
+		stringFormat := "%v='%s'"
+
+		naisFileName, err := cmd.Flags().GetString("file")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error when getting flag: file. %v\n", err)
+			os.Exit(1)
+		}
+
+		outputFormat, err := cmd.Flags().GetString("output")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error while getting flag: output. %v\n", err)
+			os.Exit(1)
+		}
+
+		zone, err := cmd.Flags().GetString("zone")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error while getting flag: zone. %v\n", err)
+			os.Exit(1)
+		}
+
+		environment, err := cmd.Flags().GetString("environment")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error while getting flag: environment. %v\n", err)
+		}
```

Comment:
should it exit here and in if below as well?

Reply:
Yep, missed that one